types/rdflib/index.d.ts

```diff
@@ -1,7 +1,8 @@
-// Type definitions for rdflib 0.17
+// Type definitions for rdflib 0.20
 // Project: http://github.com/linkeddata/rdflib.js
 // Definitions by: Cénotélie <https://github.com/cenotelie>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
```

Comment:
I added that because I'm using `unknown` in [the `callback` for `put`](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/35783/files#diff-42ffe3a9415657a8798f288f3039ef00R1247) for the response type. I could change it to `any` or a generic if supporting lower versions is more important than type safety in one instance?

Reply:
Well, there's a trade-off.  Consumers on TS >= 3.0 are getting more type safety, but consumers on TS < 3.0 won't be able to consume these type definitions at all (presumably, resulting in less type safety).  It's up to the owners of each package of type definitions to decide which is more valuable to their users.

Reply:
To be explicit, I think the PR is mergeable either way.  Let me know what you'd like to do.

Reply:
Thanks. My personal judgment would be to merge it as-is (i.e. for TS 3). The current typings are sufficiently incomplete that I don't think too many people are using them yet, and the actual package itself is not used that often yet either - so many people using this will be starting out afresh with TypeScript and are thus likely to install a recent version.